Expecto.Differ/Library.fs

```diff
@@ -0,0 +1,45 @@
+module Expecto.Differ
+open System
+open DiffPlex.DiffBuilder
+open DiffPlex.DiffBuilder.Model
+
+let colorizedDiff (colorizeText: ConsoleColor -> string -> string) first second =
```

Comment:
We use `en-GB` in this repo, so `colourisedDiff`, if you please ;)

Reply:
Let's chat about the colourisation logic soon.

Reply:
Alright, consistency is key :) 

Even though I'm a little confused by the choice of using "colour" etc. when the rest of the industry uses "color" – we're even using the ConsoleColor type here. But at the end of the day it doesn't matter of course as long as it's consistent. I'll fix!